File: server/src/main/java/com/mthwate/conk/listener/AbstractServerListener.java
package com.mthwate.conk.listener;

import com.jme3.network.HostedConnection;
import com.jme3.network.Message;
import com.mthwate.conk.message.AbstractMessageListener;

/**
 * @author mthwate
 */
public abstract class AbstractServerListener<T extends Message> extends AbstractMessageListener<HostedConnection, T> {

}

File: server/src/main/kotlin/com/mthwate/conk/listener/AbstractServerListener.kt
package com.mthwate.conk.listener

import com.jme3.network.HostedConnection
import com.jme3.network.Message
import com.mthwate.conk.message.AbstractMessageListener

/**
 * @author mthwate
 */
abstract class AbstractServerListener<T : Message> : AbstractMessageListener<HostedConnection, T>()
